release.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "Travis + Gradle release"
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
+git config --global push.default current
+git stash
+git remote set-url origin https://$GH_TOKEN@github.com/reportportal/agent-java-spock
+git checkout master
+git update-index --chmod=+x gradlew
+chmod +x gradlew
+./gradlew release -Prelease.useAutomaticVersion=true --debug --stacktrace
```

Comment:
just to make sure - will debug mode expose some security-critical info? if so, can we avoid using debug mode there?

Reply:
agree, my bad, forgot to rip it off after I tested it locally

Reply:
JFYI no exposure, this is how it is logged:
$ export BINTRAY_USER=[secure]
removed anyways